views/MainView.jsx

```diff
@@ -34,7 +34,7 @@ const MainView = ({ navigation }) => {
               );
             } else {
               return (
-                <Article article={item} key={item.id} navigation={navigation} />
+                <Article article={item} navigation={navigation} />
```

Comment:
Why did you remove key? Dosn't it raise a warning now?

Reply:
Nops, FlatList takes care of that :)